modules/tests/src/test/scala/higherkindness/mu/rpc/kafka/ServiceSpec.scala

```diff
@@ -102,25 +102,26 @@ class ServiceSpec extends AnyFunSuite with Matchers with OneInstancePerTest with
           topicName <- "topic".pure[IO]
           create    <- client.createTopic(CreateTopicRequest(topicName, 2, 1)).attempt
           _         <- IO(assert(create.isRight))
-          describe  <- client.describeTopics(DescribeTopicsRequest(List(topicName))).attempt
```

Comment:
ℹ️  I had to comment on some of the assertions in this file, as they fail after upgrading the `embedded-kafka `version. It'll need to be fixed in a separate PR.

Reply:
I think we can remove that module altogether, no?